### PR TITLE
chore: Add StackBlitz template

### PR DESCRIPTION
**Pull request by yusufkandemir**
**What kind of change does this PR introduce?**

- Documentation
- Issue form update

**Does this PR introduce a breaking change?**

- No

**The PR fulfills these requirements:**

- It's submitted to the `dev` branch (or `v[X]` branch)

**Other information:**

### CODE REVIEW

**Comment by smolinari** [comment]
From what I can tell... LGTM! 

Scott

File: docs/src/components/HeaderMenu.vue [hunk 1]
```diff
@@ -107,6 +107,16 @@ div
           q-icon(:name="fabJsfiddle")
         q-item-section jsFiddle
 
+      q-item(clickable, href="https://stackblitz.com/fork/quasarframework", target="_blank", rel="noopener")
+        q-item-section.text-black(avatar)
+          q-icon(:name="fasBolt")
```
Comment (ldiebold) [inline docs/src/components/HeaderMenu.vue:112]: Maybe we could make these blue? `color="blue"`

**Comment by ldiebold** [comment]
This looks great!
made a quick comment about possibly making the menu icons blue:
image

**Comment by IlCallo** [comment]
LGTM too, I agree with Luke about setting the stackblitz icon color to blue

File: docs/src/components/HeaderMenu.vue (hunk 1 above)
Comment (yusufkandemir) [reply docs/src/components/HeaderMenu.vue:112]: Nice suggestion, I went with `light-blue-14`.

**Pull request merged**